rewrite-javascript/rewrite/tsconfig.build.json

```diff
@@ -8,7 +8,6 @@
     "removeComments": true
   },
   "include": [
-    "src/**/*",
-    "test/**/*"
+    "src/**/*"
```

Comment:
callout: the `tsc` compiler isn't very smart, so what this was resulting in is the `test/**` sources being built twice which isn't desirable.

Reply:
We'd still need some more refactoring to get rid of this. The problem is that some integration tests fail without this. The tests aren't executed on CI, so I can also fix this afterwards.

Reply:
Just a clarification, but as in the Java integration tests?

Reply:
Yes. They depend on a JS recipe which is in test. I will try to fix that.